<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1493</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USDB Syncer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,10,0,2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>UltraStar Song Database (usdb)</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_get_songlist">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Refresh song list</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/resources.qrc">
             <normaloff>:/icons/faviconUSDB.ico</normaloff>:/icons/faviconUSDB.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>347</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Synchronize to local song directory</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_song_dir">
           <property name="text">
            <string>./songs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_select_song_dir">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Select song folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Search (RexExp)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="5,1,0">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_search"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_search_column">
         <item>
          <property name="text">
           <string>any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Artist</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/artist.png</normaloff>:/icons/artist.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Title</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/title.png</normaloff>:/icons/title.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Language</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/language.png</normaloff>:/icons/language.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Edition</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/edition.png</normaloff>:/icons/edition.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_case_sensitive">
         <property name="text">
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Filter (not implemented yet)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,2,0,2,0,2,0,2,0,1,0,1,0,1">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/artist.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_artist">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/title.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_title">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/language.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_language">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/edition.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_edition">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/golden_notes.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_goldennotes">
         <item>
          <property name="text">
           <string>any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>no</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/rating.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_rating">
         <item>
          <property name="text">
           <string>any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜…</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜…â˜…</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜…â˜…â˜…</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜…â˜…â˜…â˜…</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜… or more</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜… or more</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜…â˜… or more</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>â˜…â˜…â˜…â˜… or more</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources/resources.qrc">:/icons/views.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_views">
         <item>
          <property name="text">
           <string>any</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>300+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>400+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500+</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView_availableSongs">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>24</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_songfile">
        <property name="title">
         <string>Download song file</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Encoding</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_encoding"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Line Endings</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_line_endings"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_audio">
        <property name="title">
         <string>Download audio</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Audio Format</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_audio_format"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="toolTip">
            <string>Login to Youtube in your favorite browser in order to access age-restricted content</string>
           </property>
           <property name="text">
            <string>Cookies</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_browser"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_reencode_audio">
        <property name="title">
         <string>Reencode audio</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_6">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Audio Format</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_audio_conversion_format"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_video">
        <property name="title">
         <string>Download video</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Video container</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_videocontainer"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Max. resolution</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_videoresolution"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Max. fps</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_fps"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_reencode_video">
        <property name="title">
         <string>Reencode video</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>25</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Video Format</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_videoencoder">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>23</y>
           <width>118</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_cover">
          <property name="title">
           <string>Download cover â€“ settings:</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_background">
          <property name="title">
           <string>Download background â€“ settings:</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Download</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_background">
             <item>
              <property name="text">
               <string>always</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>only if no video</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="toolButton_infos">
             <property name="toolTip">
              <string>Information</string>
             </property>
             <property name="text">
              <string>Infos</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_warnings">
             <property name="toolTip">
              <string>Warnings</string>
             </property>
             <property name="text">
              <string>Warnings</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/warning.png</normaloff>:/icons/warning.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_errors">
             <property name="toolTip">
              <string>Errors</string>
             </property>
             <property name="text">
              <string>Errors</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/error.png</normaloff>:/icons/error.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_downloadSelectedSongs">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Download selected songs!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1493</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_meta_tags"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_meta_tags">
   <property name="text">
    <string>Create Meta Tags</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
